commodore/component-template/{{ cookiecutter.slug }}/Makefile.vars.mk

```diff
@@ -35,6 +35,16 @@ COMMODORE_CMD  ?= $(DOCKER_CMD) $(DOCKER_ARGS) $(root_volume) docker.io/projects
 COMPILE_CMD    ?= $(COMMODORE_CMD) component compile . $(commodore_args)
 JB_CMD         ?= $(DOCKER_CMD) $(DOCKER_ARGS) --entrypoint /usr/local/bin/jb docker.io/projectsyn/commodore:latest install
 
+{%- if cookiecutter.add_golden == "y" %}
+GOLDEN_FILES    ?= $(shell find tests/golden/$(instance) -type f)
+
+KUBENT_FILES    ?= $(shell echo "$(GOLDEN_FILES)" | sed 's/ /,/g')
```

Comment:
If I recall correctly, the length of a shell command is limited. Depending on the amount of golden files, that limit can be reached, can't it?

Reply:
The length might be reached in some extreme cases, but it's imho unlikely, since modern Linux supports up to 128KiB **per argument** and up to 2MiB total for a command invocation.

We could just pass the files to kubent with `-f $(find tests/golden/$(instance) -type f -printf '%p,' | sed 's/,$//')` instead of precomputing the list in a make variable. I just reused the same pattern which we use for other make targets.

Reply:
Na, I am fine with having it in the variable.